documents/order.md

```diff
@@ -0,0 +1,219 @@
+## Orders
+
+### Create order
+
+```go
+data := map[string]interface{}{
+  "amount": 50000,
+  "currency": "INR",
+  "receipt": "some_receipt_id",
+  "payment_capture": 1,
```

Comment:
The public docs says that payment_capture is deprecated. https://razorpay.com/docs/api/orders/#orders-api
Can see even partial_payment in the input params, please add that too.